runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -39,22 +37,22 @@
  * In the Corfu runtime, on top of a stream,
  * an SMR object layer implements objects whose history of updates
  * are backed by a stream.
- *
+ * <p>
```

Comment:
This seems to misformat the javadoc comment needlessly?

Reply:
hmm, intellij did that. Removed.